src/postprocess/DOS.jl

```diff
@@ -9,6 +9,39 @@
 
 using ForwardDiff
 
+@doc raw"""
+    NOS(ε, basis, orben; smearing=basis.model.smearing, T=basis.model.temperature)
+
+The number of Kohn-Sham states in a temperature window of width T around the energy ε
+contributing to the DOS at temperature T.
+
+This quantity is not a physical quantity, much rather a dimensionaless approximate measure
+for how well properties near the Fermi surface are sampled with the passed `smearing`
+and temperature `T`. It increases with both `T` and better sampling of the BZ with
+``k``-Points. A value ``\gg 1`` indicates a good sampling of properties near the
+Fermi surface.
+"""
+function NOS(ε, basis, orben; smearing=basis.model.smearing, T=basis.model.temperature)
+    N = zero(ε)
+    T != 0 || error("NOS only supports finite temperature")
+
+    # Note the differences to the DOS and LDOS functions: We are not counting states
+    # per BZ volume (like in DOS), but absolute number of states. Therefore n_symeqk
+    # is used instead of kweigths. The number of states is counted over a temperature
+    # window of width T, where we assume that the number of states remains constant
+    # in the interval [T - T/2, T + T/2] as ( d/dε f_T(εik - ε') )|_{ε' = ε}
+    # with f_T the smearing function. This gives the factor T in front.
```

Comment:
I'm still confused by this description, sorry if I'm being dense

Reply:
No that's fine. Maybe it's my picture of the situation, which is wrong. I tried to detail it again.